cpp/src/robotick/framework/data/RemoteEngineConnection.cpp

```
@@ -380,7 +399,22 @@ namespace robotick
 		set_state(State::ReadyForHandshake);
 	}
 
-	void RemoteEngineConnection::tick_sender_send_handshake()
+	static inline uint32_t float_to_network_bytes(float value)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)